docs/getting-started.asciidoc

```diff
@@ -0,0 +1,224 @@
+
+[float]
+[[tutorial-define-index]]
+=== Defining Your Index Patterns
+
+Each set of data loaded to Elasticsearch has an https://www.elastic.co/guide/en/kibana/current/settings.html#settings-create-pattern[index pattern]. In the previous section, the Shakespeare data set has an index named `shakespeare`, and the accounts 
+data set has an index named `bank`. An _index pattern_ is a regular expression that can 
+match multiple indices. For example, in the common logging use case, a typical index name contains the date in MM-DD-YYYY 
+format, and an index pattern for May would look something like `logstash-05-*`.
+
+For this tutorial, any pattern that matches either of the two indices we've loaded will work. Open a browser and 
+navigate to `localhost:5601`. Click the *Settings* tab, then the *Indices* tab. Click *Add New* to define a new index 
+pattern. Since these data sets don't contain time-series data, make sure the *Index contains time-based events* box is 
+unchecked. Specify `shakes*`  as the index pattern for the Shakespeare data set and click *Create* to define the index 
+pattern, then define a second index pattern named `ba*`.
+
+[float]
+[[tutorial-discovering]]
+=== Discovering Your Data
+
+Click the *Discover* tab to display Kibana's data discovery functions:
+
+image::images/tutorial-discover.png[]
+
+Right under the tab itself, there is a search box where you can search your data. Searches take a specific 
+{ref}/query-dsl-query-string-query.html#query-string-syntax[query syntax] that enable you to create custom searches, 
+which you can save and load by clicking the buttons to the right of the search box.
+
+Beneath the search box, the current index pattern is displayed in a drop-down. You can change the index pattern by 
+selecting a different pattern from the drop-down selector.
+
+Try selecting the `ba*` index pattern and putting the following search into the search box:
+
+[source,text]
+account_number:<100 AND balance:>47500
+
+If you're using the linked sample data set, this search returns 5 results: Account numbers 8, 32, 78, 85, and 97.
+
+image::images/tutorial-discover-2.png[]
+
+To narrow the display to only the specific fields of interest, highlight each field in the list that displays under the 
+index pattern and click the *Add* button. Note how, in this example, adding the `account_number` field changes the 
+display from the full text of five records to a simple list of five account numbers:
+
+image::images/tutorial-discover-3.png[]
+
+[float]
+[[tutorial-visualizing]]
+=== Data Visualization: Beyond Discovery
+
+The visualization tools available on the *Visualize* tab enable you to display aspects of your data sets in several 
+different ways. Visualizations depend on Elasticsearch {ref}/search-aggregations.html[aggregations] in two different 
+types: _bucket_ aggregations and _metric_ aggregations. A bucket aggregation sorts your data according to criteria you 
```

Comment:
Feels like metrics and buckets would be better explained further down, once they're properly in the visualization builder.